src/Microsoft.Azure.AppConfiguration.IsolatedAzureFunctions/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.AppConfiguration.IsolatedAzureFunctions
+{
+    /// <summary>
+    /// Middleware for Azure App Configuration to use activity-based refresh for key-values registered in the provider.
+    /// </summary>
+    internal class AzureAppConfigurationRefreshMiddleware : IFunctionsWorkerMiddleware
+    {
+        public IEnumerable<IConfigurationRefresher> Refreshers { get; }
```

Comment:
This can be a private field. No need to be public property.

Reply:
Never mind. Your tests need it.

Reply:
I don't like the idea of modifying class design to satisfy tests. With more occurrences it can quickly get messy. I think we can accomplish the same test while keeping this private.

Reply:
The tests are only verifying if the middleware gets a reference to all instances of `IConfigurationRefresher` from an `IConfigurationRefresherProvider`. If `Refreshers` field is private, how do we verify that in the tests?
Also, the `AzureAppConfigurationRefreshMiddleware` class is still internal. So I think making the `Refreshers` field public should be okay.

Reply:
> Also, the AzureAppConfigurationRefreshMiddleware class is still internal. So I think making the Refreshers field public should be okay.

It's okay in the sense of it not being exposed as a part of the public API of the package. But it detracts from the design of the class. 

A way to perform the same test without exposing Refreshers is to pass an implementation of IConfigurationRefresher is to call AzureAppConfigurationRefreshMiddleware.Invoke and make sure each Refresher was called.

Reply:
Updated, thanks!